lib/NativeCode/DynamicLibraryLoaderHelper/Directory.Build.props

```diff
@@ -15,12 +15,19 @@
     <NormalizedPlatform Condition="'$(Platform)' == 'x86' or '$(Platform)' == 'Win32'">x86</NormalizedPlatform>
     <NormalizedPlatform Condition="'$(Platform)' == 'x64'">x64</NormalizedPlatform>
     
-    <!-- 
+    <EOS_SDK_DLL_NAME Condition="'$(NormalizedPlatform)' == 'x64'">EOSSDK-Win64-Shipping.dll</EOS_SDK_DLL_NAME>
+    <STEAM_API_DLL_NAME Condition="'$(NormalizedPlatform)' == 'x64'">steam_api64.dll</STEAM_API_DLL_NAME>
+
+    <EOS_SDK_DLL_NAME Condition="'$(NormalizedPlatform)' == 'x86'">EOSSDK-Win32-Shipping.dll</EOS_SDK_DLL_NAME>
+    <STEAM_API_DLL_NAME Condition="'$(NormalizedPlatform)' == 'x86'">steam_api.dll</STEAM_API_DLL_NAME>
+
+      <!-- 
       Define a property with the SolutionDir path but with backslashes replaced 
       by forward slashes 
       -->
     <SolutionDirForwardSlashes>$(SolutionDir.Replace('\', '/'))</SolutionDirForwardSlashes>
     <UnityStreamingAssetsDirectory>$(SolutionDirForwardSlashes)../../../Assets/StreamingAssets/</UnityStreamingAssetsDirectory>
-    <OutputUnityAssetsDirectory>$(SolutionDir)..\..\..\Assets\Plugins\Windows\$(NormalizedPlatform)\</OutputUnityAssetsDirectory>
+    <SteamLibraryDirectory>$(SolutionDirForwardSlashes)../../../Library/PackageCache/com.rlabrecque.steamworks.net*/Plugins/</SteamLibraryDirectory>
```

Comment:
It seems strange to reach into the /Library folder for things. If I have a clean build of the repo, and go to Native code first, will this still correctly build? Or is this the output directory?

Reply:
This is only used in debug mode. What's happening is that the console application that can be used to run all the native code without unity needs access to the `steam_api64.dll` (or `steam_api.dll`) file. 

When making a build from within Unity - the appropriate file is placed in the built output directory. When making a test build that just checks to see if the code that hooks into the dll is working - it needs to know where that file is.

In summary - it is strange - and if you're building a debug build of the native code from a fresh repository it will still succeed - but will not be able to execute the component relating to steam integration. It will behave as if there is no steam integration - which is actually how we have it in our project, so it would behave as expected.